charts/schema-registry/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "schema-registry.configmap" . }}
+data:
+  "CONTROLLER_URL": "tcp://{{ .Values.controller.ip }}:{{ .Values.controller.port }}"
```

Comment:
When TLS is enabled, the scheme is no more `tcp://` -it just be `tls://`. Also, we should handle discovery URLs like `pravega://` and `pravegas://`.  

https://github.com/pravega/pravega/blob/c7ac009970787df2633163644691aa3d72fc1a4e/client/src/main/java/io/pravega/client/stream/impl/ControllerResolverFactory.java#L62